airbyte_cdk/test/standard_tests/docker_base.py

```
@@ -0,0 +1,191 @@
+        """Get the file path that contains the class."""
+        module = sys.modules[cls.__module__]
+        # Get the directory containing the test file
+        return Path(inspect.getfile(module)).parent
+
+    @classmethod
+    def get_connector_root_dir(cls) -> Path:
+        """Get the root directory of the connector."""
+        return find_connector_root([cls.get_test_class_dir(), Path.cwd()])
+
+    @classproperty
+    def acceptance_test_config_path(cls) -> Path:
```

Pylint (3.3.7)

[error] Method 'acceptance_test_config_path' should have "self" as first argument

(E0213)